Create DRSRule.md

lucdekens opened the pull request:
Add the DRSRule module

lucdekens committed:
Create DRSRule.md

Add the DRSRule module

ClaudioESSilva commented on Modules/DRSRule.md:
```diff
@@ -0,0 +1,17 @@
+
+| Works with | Author(s)  | Project URL(s) | Last Update   | Tags |
+|------------|------------|----------------|---------------|------|
+| vSphere 5+ | Matt Boren | [GitHub](https://https://github.com/PowerCLIGoodies/DRSRule) | May 2016 | vSphere, Cluster, DRS, Rules, Groups |
```
The URL has http:// twice
Please add both authors name on the same cell and if you don't mind if you want, you should add the b | t with links for each author

ClaudioESSilva commented:
Thank you @lucdekens for your contribution. Please see the comments.

lucdekens committed:
Update DRSRule.md

Removed double http entry
Added Blog & Twitter info for both authors
Authors in one cell

lucdekens committed:
DRSRule.md Update

Rearranged Shared blog posts

ClaudioESSilva commented on Modules/DRSRule.md:
```diff
@@ -0,0 +1,18 @@
+
+### Shared blog posts:
+Matt Boren (<a href="http://www.vnugglets.com" target="_blank">b</a> \| <a href="https://twitter.com/mtboren" target="_blank">t</a>)
+- [DRSRule – a DRS rules and groups module](http://www.lucd.info/2015/01/22/drsrule-drs-rules-and-groups-module/) @ 2015-01-22
```
I think the blog posts need to be switch. This one is yours and the other under your name is from Matt.
Everything eles seems ok! 😄

ClaudioESSilva reviewed (changes requested):
The blog entries are under the wrong author

lucdekens committed:
Update DRSRule.md

Switched shared blog posts

ClaudioESSilva commented:
Thanks for your contribution @lucdekens .
Looking forward for more 😄
